iron-list.html

```diff
@@ -908,6 +932,27 @@
 
     _scrollHandler: function() {
       this._refresh();
+
+      this.debounce('scrollThreshold', function() {

```

Comment:
Any reason this needs debouncing?  Since this is only called from a `scroll` event and (to my knowledge) you can't get a second scroll event in the same microtask, not sure this adds anything?